.github/workflows/pitest.yml

```diff
@@ -7,8 +7,18 @@ on:
   pull_request:
 
 jobs:
-  pitest-annotation:
+  pitest:
+    strategy:
+      matrix:
+        profile: [ pitest-annotation, pitest-ant, pitest-api, pitest-blocks, pitest-coding-1,
+                   pitest-coding-2,pitest-coding-require-this-check, pitest-common,
+                   pitest-common-2, pitest-design, pitest-filters, pitest-header, pitest-imports,
+                   pitest-indentation, pitest-javadoc, pitest-main, pitest-metrics, pitest-misc,
+                   pitest-modifier, pitest-naming, pitest-packagenamesloader, pitest-regexp,
+                   pitest-sizes, pitest-tree-walker, pitest-utils, pitest-whitespace,
+                   pitest-xpath, pitest-java-ast-visitor ]
```

Comment:
Please do single line array declaration, instead of square bracket array declaration. Example:

```yml
profile:
    - pitest-annotation
    - pitest-ant
    - ...

```
This will make for less diff in future and easier to see each profile.

Reply:
Done.